src/ai/backend/client/func/acl.py

```diff
@@ -0,0 +1,36 @@
+import textwrap
+from typing import Sequence
+
+from ai.backend.client.output.fields import permission_fields
+from ai.backend.client.output.types import FieldSpec
+
+from ..session import api_session
+from .base import BaseFunction, api_function
+
+__all__ = ("Permission",)
+
+_default_list_fields = (permission_fields["vfolder_host_permission_list"],)
+
+
+class Permission(BaseFunction):
+    @api_function
+    @classmethod
+    async def list(
+        cls,
+        fields: Sequence[FieldSpec] = _default_list_fields,
+    ) -> Sequence[str]:
+        """
+        Fetches the list of atomic permissions.
+
+        :param fields: Additional permission query fields to fetch.
+        """
+        query = textwrap.dedent(
+            """\
+            query {
+                permissions {$fields}
```

Comment:
According to the query schema defined [here](https://github.com/lablup/backend.ai/pull/790/files#diff-56a2d9f1fc07827156313028f4414a518eb3f14bc56739d16d6bbb3e32956019R490-R493), the query name should be changed to "vfolder_host_permission", not "permissions".